### PR TITLE
Fix issues in the Console app

### PR DESCRIPTION
**Pull request by thivi**

### CODE REVIEW

File: apps/console/src/features/groups/components/wizard/group-basics.tsx [hunk 1]
```diff
@@ -86,35 +78,15 @@ export const GroupBasics: FunctionComponent<GroupBasicProps> = (props: GroupBasi
      * TODO : Discuss and add or remove the Hybrid domains
      *        to the dropdown.
      */
-    const groupDomains = [{
+    const groupDomains = [ {
         key: -1, text: SharedUserStoreConstants.PRIMARY_USER_STORE, value: SharedUserStoreConstants.PRIMARY_USER_STORE
```
Comment (brionmario) [inline apps/console/src/features/groups/components/wizard/group-basics.tsx:82]: Possible max line issue here. If so better to format.

```suggestion
        key: -1,
        text: SharedUserStoreConstants.PRIMARY_USER_STORE,
        value: SharedUserStoreConstants.PRIMARY_USER_STORE
```

File: apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx [hunk 2]
```diff
@@ -127,8 +128,48 @@ const DynamicConnectorForm = (props) => {
     );
 };
 
+const validate = (values) => {
+    const errors = {};
+    if (
+        parseInt(
```
Comment (brionmario) [inline apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx:134]: Would be better to extract this to a variable for more readability. Check other places as well.

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx (hunk 2 above)
Comment (thivi) [reply apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx]: Resolved by 601c7fa

File: apps/console/src/features/groups/components/wizard/group-basics.tsx (hunk 1 above)
Comment (thivi) [reply apps/console/src/features/groups/components/wizard/group-basics.tsx]: Resolved by 97e62f1

File: apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx [hunk 3]
```diff
@@ -127,8 +128,44 @@ const DynamicConnectorForm = (props) => {
...
+    const allowedIdleTimeSpanName = GovernanceConnectorUtils.encodeConnectorPropertyName(
+        ServerConfigurationsConstants.ALLOWED_IDLE_TIME_SPAN_IN_DAYS
+    );
+
+    const allowedIdleTimeSpan = parseInt(values[ allowedIdleTimeSpanName ]);
+
+    if (allowedIdleTimeSpan < 0) {
+        errors[ allowedIdleTimeSpanName ]
+            = I18n.instance.t("adminPortal:components.governanceConnectors.form.errors.positiveIntegers");
+
```
Comment (jeradrutnam) [inline apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx:143]:
```suggestion
```

File: apps/console/src/features/server-configurations/constants/server-configurations-constants.ts [hunk 4]
```diff
@@ -213,4 +213,8 @@ export class ServerConfigurationsConstants {
 		"code while updating the configurations.";
 	public static readonly CONFIGS_UPDATE_REQUEST_ERROR: string = "An error occurred while updating the " +
 		"configurations.";
+
+	//Idle account suspend names
```
Comment (jeradrutnam) [inline apps/console/src/features/server-configurations/constants/server-configurations-constants.ts:217]:
```suggestion
	// Idle account suspend names
```

File: apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx [hunk 5]
```diff
@@ -127,8 +128,44 @@ const DynamicConnectorForm = (props) => {
...
+    }
+
+    if (
+        !RegExp(/^(?:\d+,)+\d+$/).test(
+            values[
+            GovernanceConnectorUtils.encodeConnectorPropertyName(
+                ServerConfigurationsConstants.ALERT_SENDING_TIME_PERIODS_IN_DAYS
+            )
```
Comment (jeradrutnam) [inline apps/console/src/features/server-configurations/components/governance-connectors/dynamic-connector-form.tsx:151]:
```suggestion
                GovernanceConnectorUtils.encodeConnectorPropertyName(
                    ServerConfigurationsConstants.ALERT_SENDING_TIME_PERIODS_IN_DAYS
                )
```

**Pull request merged**